backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/VmInfoBuildUtils.java

```diff
@@ -1740,4 +1740,9 @@ public String getVdsmCpuPinningPolicy(VM vm) {
         }
         return cpuPinningPolicyName;
     }
+
+    public static boolean needsMemtune(VM vm) {
+        return vm.getBiosType() != null && vm.getBiosType().getChipsetType() == ChipsetType.Q35
```

Comment:
this bug is regarding VFIO although here we just check the Q35 and CPUs. do we need to check if we have mdev on the domain at all?

Reply:
It depends on how much specific we want to be.  In the most elaborate case we could count VFIO devices and introduce some computation of the limit based on that.  Checking for their presence would be a similar effort and would complicate the patch.  I'm currently not aware about any real problem that could be caused by adding the limit so I suggested this simple method of setting it for any large q35 VM (in the terms of vCPUs) and have been waiting whether there are any objections. :-)

Reply:
OK, if nothing is harmful here :)